### PR TITLE
feat(makefile): add packagemanifest support

### PR DESCRIPTION
**Pull request by georgettica**

### CODE REVIEW

**Comment by georgettica** [comment]
/hold
removing from WIP so don't want it to go straight through

File: hack/app_sre_create_image_catalog.sh [hunk 1]
```diff
@@ -64,30 +53,31 @@ fi
...
-    # stopping script as that version was already built, so no need to rebuild it
-    exit 0
-fi
+# Build an image locally that has all tools we need
+#
+# TODO: This section is done as running stuff on the jenkins slave
+#       the doesn't have `operator-sdk >=1` and `kustomize` to resort
+#       to building a container that we do all operations in
+#
+# what this section do in general is `make packagemanifests`
```
Comment (NautiluX) [inline hack/app_sre_create_image_catalog.sh:65]:
```suggestion
# TODO: This section is done as in the pipeline stuff is running on the Jenkins slave
#       that doesn't have `operator-sdk >=1` and `kustomize`
#       so we build a container with those tools and run all operations inside
#
# what this section does in general is `make packagemanifests`
```

File: hack/app_sre_create_image_catalog.sh [hunk 2]
```diff
@@ -98,24 +88,21 @@ replaces $PREV_VERSION
...
-RUN initializer --permissive
 
-CMD ["registry-server", "-t", "/tmp/terminate.log"]
-EOF
+pushd $SAAS_OPERATOR_DIR
+  git push origin "$BRANCH_CHANNEL"
+popd
 
-docker build -f $DOCKERFILE_REGISTRY --tag "${REGISTRY_IMG}:${BRANCH_CHANNEL}-latest" .
+# add, commit & push
```
Comment (NautiluX) [inline hack/app_sre_create_image_catalog.sh:105]:
```suggestion
```

**Review by NautiluX — CHANGES_REQUESTED** [review]
suggested some small changes to your comments

**Comment by georgettica** [comment]
/unhold
Let it rip!

**Review by NautiluX — APPROVED** [review]
/lgtm

**Pull request merged**